app/helpers/email_blocklist_helper.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+module EmailBlocklistHelper
```

Comment:
Rename to `EmailBlocklistValidation` and `extend ActiveSupport::Concern` (look at `app/models/concerns/token.rb` which is quite similar), then move the model `validate` call and validation method into this module.

If we need the method elsewhere, I'm OK with a call saying `EmailBlocklistValidation.email_on_blacklist?` referring directly to the object from the global namespace, I don't think the OO style is improved by mixing it into more objects. But having a mixin would let us remove the duplication in these models!